src/core/Akka/Actor/Settings.cs

```diff
@@ -111,6 +111,18 @@ public Settings(ActorSystem system, Config config, ActorSystemSetup setup)
 
             LogLevel = Config.GetString("akka.loglevel", null);
             StdoutLogLevel = Config.GetString("akka.stdout-loglevel", null);
+            
+            // FILTER MUST ALWAYS BE LOADED BEFORE STANDARD OUT LOGGER
+            // check to see if we have a LogFilterSetup in the ActorSystemSetup
+            var logFilterSetup = Setup.Get<LogFilterSetup>();
```

Comment:
Load the `LogFilterSetup` safely from the `ActorSystemSetup`. **There will be no HOCON equivalent for loading this**.